test/parallel/test-child-process-exec-abortcontroller-promisified.js

```diff
@@ -10,12 +10,9 @@ const invalidArgTypeError = {
   name: 'TypeError'
 };
 
-let waitCommand = '';
-if (common.isWindows) {
-  waitCommand = 'TIMEOUT 120';
-} else {
-  waitCommand = 'sleep 2m';
-}
+const waitCommand = common.isLinux ?
```

Comment:
If there's a risk of killing the child process not working (https://github.com/nodejs/node/issues/37568#issuecomment-788922748) on other POSIX platforms it may be safer to only use the Node.js variant on Windows only.

Reply:
My reasoning is we should always rely on Node.js when we want to test a cross-platform behaviour – we should avoid relying on `sleep` being cross-platform consistent. Here Linux being the odd one out (https://github.com/nodejs/node/issues/37518), it needs special treatment.

That being said, if the consensus is to go the other way around, that's fine with me.

Reply:
We're now seeing the Linux problem on SmartOS so I've opened https://github.com/nodejs/node/pull/46276 to swap things back to using `sleep` on all POSIX platforms. https://github.com/nodejs/node/issues/37518 covers the issue.